jaxrs-tck/src/main/java/jakarta/ws/rs/tck/ee/rs/client/asyncinvoker/JAXRSClientIT.java

```diff
@@ -2701,7 +2701,7 @@ public void putWithCallbackThrowsNoWebApplicationExceptionForResponseTest()
    * @test_Strategy: Invoke HTTP trace method for the current request
    * asynchronously.
    */
-  @Test
+  //@Test
```

Comment:
Wouldn't it be better to use the `@Disabled` annotation?

Reply:
I suppose @Disabled is the right way for disabling a test. Considering these tests will need to be fixed to run, I think it should be fine to comment out this way for now.  Thanks.

Reply:
@jelemux Does this answer your question?